server/people.js

```diff
@@ -37,6 +37,15 @@ const deletePerson = async (client, id) => {
   return result;
 };
 
+const changeAdmin = async (client, id) => {
```

Comment:
Instead of toggling admin, we should set it to a definite true or false. That way the UI can never be out of sync with the database. That means we'll need to pass in a third argument (admin) and then do `'UPDATE "Dootman" SET admin = $2 WHERE id = $1', [id, admin]`